.github/workflows/CD.yml

```diff
@@ -9,10 +9,9 @@ on:
 jobs:
   Release:
     name: Release XCFramework
-    runs-on: macos-14
+    runs-on: macos-15
     env:
       CODESIGN_KEY_BASE64: "${{ secrets.CODESIGN_KEY_BASE64 }}"
-      DEVELOPER_DIR: /Applications/Xcode_15.2.app
```

Comment:
Can we leave this `Xcode_16.0.app` here ?

GitHub action will always bump new Xcode versions and added/removed new simulators, which is not really stable

Because SDWebImage some test case check current OS (like iPhone simulator on iOS 18.0) behavior, which may be buggy and changed by Apple when they release new minor versions. (like Xcode 15.0->15.1)

Reply:
done